From 5fa03879b348058ff1474dd257494c03050141ed Mon Sep 17 00:00:00 2001
From: Richard Zhu <Richard.Zhu@freescale.com>
Date: Fri, 13 Mar 2015 16:24:11 +0800
Subject: [PATCH 0480/5242] MLK-10466-2 ARM: imx: add the pcie related macros
 definitions

commit  4a3af7369ab2172840e901cef170471366731a7b from
https://source.codeaurora.org/external/imx/linux-imx.git

add the pcie related macros definitions into gpr.

Signed-off-by: Richard Zhu <Richard.Zhu@freescale.com>
(cherry picked from commit b4a5b2e53b2e743824d0af7428f7d9d406bec8bd)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 include/linux/mfd/syscon/imx6q-iomuxc-gpr.h |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/linux/mfd/syscon/imx6q-iomuxc-gpr.h b/include/linux/mfd/syscon/imx6q-iomuxc-gpr.h
index f38ee27..56f6c97 100644
--- a/include/linux/mfd/syscon/imx6q-iomuxc-gpr.h
+++ b/include/linux/mfd/syscon/imx6q-iomuxc-gpr.h
@@ -101,6 +101,7 @@
 #define IMX6Q_GPR1_PCIE_ENTER_L1		BIT(26)
 #define IMX6Q_GPR1_MIPI_COLOR_SW		BIT(25)
 #define IMX6Q_GPR1_DPI_OFF			BIT(24)
+#define IMX6Q_GPR1_PCIE_SW_PERST		BIT(23)
 #define IMX6Q_GPR1_EXC_MON_MASK			BIT(22)
 #define IMX6Q_GPR1_EXC_MON_OKAY			0x0
 #define IMX6Q_GPR1_EXC_MON_SLVE			BIT(22)
@@ -319,6 +320,7 @@
 #define IMX6Q_GPR12_ARMP_AHB_CLK_EN		BIT(26)
 #define IMX6Q_GPR12_ARMP_ATB_CLK_EN		BIT(25)
 #define IMX6Q_GPR12_ARMP_APB_CLK_EN		BIT(24)
+#define IMX6Q_GPR12_PCIE_PM_TURN_OFF		BIT(16)
 #define IMX6Q_GPR12_DEVICE_TYPE			(0xf << 12)
 #define IMX6Q_GPR12_PCIE_CTL_2			BIT(10)
 #define IMX6Q_GPR12_LOS_LEVEL			(0x1f << 4)
-- 
1.7.9.5

